src/main/java/ama/post/PostController.java

```diff
@@ -120,11 +128,16 @@ public String downvoteComment(HttpServletRequest request, @RequestParam("id") St
     public String replyToComment(HttpServletRequest request, @PathVariable(value="id") String id, String reply) {
         CommentPost context = commentPostRepository.findById(Long.parseLong(id));
         User user = authenticator.getCurrentUser();
-        if (user != null && context != null && reply != null && reply != "") {
-            CommentPost replyPost = new CommentPost(user, context, reply);
-            commentPostRepository.save(replyPost);
+        if (context != null) {
+            SubmissionPost post = context.getSubmissionPost();
```

Comment:
Oh yes. I forgot that users cannot reply to question answers. I added this in the case that comment chains were a thing... will remove.